server/resource/kurtosis_package_indexer_resource.go

```diff
@@ -12,6 +14,17 @@ func NewKurtosisPackageIndexer() *KurtosisPackageIndexer {
 	return &KurtosisPackageIndexer{}
 }
 
-func (resource KurtosisPackageIndexer) Ping(ctx context.Context, ping *generated.IndexerPing) (*generated.IndexerPong, error) {
+func (resource *KurtosisPackageIndexer) Ping(_ context.Context, _ *generated.IndexerPing) (*generated.IndexerPong, error) {
 	return &generated.IndexerPong{}, nil
 }
+
+func (resource *KurtosisPackageIndexer) GetPackages(_ context.Context, _ *emptypb.Empty) (*generated.GetPackagesResponse, error) {
+	// TODO: implement
+	hyperlanePackage := api_constructors.NewKurtosisPackage(
+		"github.com/kurtosis-tech/hyperlane-package",
```

Comment:
looks good; so untyped args corresponds to yaml/json - maybe just add the comment. everything looks good. also for the server we we would be using `connect-go` .